pb/edge_ctrl_pb/edge_ctrl.proto

```diff
@@ -109,6 +111,46 @@ message Service {
   bool encryptionRequired = 3;
 }
 
```

Comment:
I see some of these types used, but not all. Are some of them for future work or intermediary stuff that could be tidied?

Reply:
They are for follow-up functionality on a different branch.